view/base/web/error-handler.js

```diff
@@ -26,6 +26,43 @@
         sendTimer = window.setTimeout(sendError, sendDelay);
     };
 
+
+    
+    // Override console.error method so these errors are also logged
+
+    var originalConsole = {
+        error: console.error
+    };
+
+
+    function logToQueue(type, args) {
+        if (sendQueue.length > 100) {
+            // That's a lot of errors! Let's not overwhelm the server with more.
+            return;
+        }
+        if (!sendQueue.length) {
+            sendTimer = window.setTimeout(sendError, sendDelay);
+        }
```

Comment:
Can we perhaps move this into a shared function instead of duplicating this logic?

Maybe we can pass in `{ colno: event.colno, filename: event.filename, lineno: event.lineno, message: event.message, stack: event.error && event.error.stack }` in the original and `{ message: Array.from(args).join(' ') }` (or whatever is appropriate here) in this new feature, and then the shared function can add the `timer` key and send this as the `event` object to the `sendQueue.push()` call?